src/main/java/optimus/Main.java

```diff
@@ -0,0 +1,54 @@
+package optimus;
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Optimus using FXML.
+ * Adapted from CS2103T JavaFX tutorial -reused
+ */
+public class Main extends Application {
+
```

Comment:
maybe can remove this line here. not sure if there is supposed to be a line here